core/src/state.cpp

```diff
@@ -40,6 +40,7 @@ void actions::push_capslock(bool turnOn) {
 state::state(km::core::abstract_processor & ap, km_core_option_item const *env)
   : _processor(ap)
 {
+  _app_ctxt.has_markers = false;
```

Comment:
explain why here?

Reply:
```suggestion
  // The app context will never have markers, because it is an exact
  // copy of the context passed in from the application, in whatever
  // normalization form that the app is using.
  _app_ctxt.has_markers = false;
```